src/gui/tray/activitylistmodel.cpp

```diff
@@ -343,6 +347,48 @@ void ActivityListModel::activitiesReceived(const QJsonDocument &json, int status
         a._dateTime = QDateTime::fromString(json.value("datetime").toString(), Qt::ISODate);
         a._icon = json.value("icon").toString();
 
+        auto richSubjectData = json.value("subject_rich").toArray();
```

Comment:
@claucambra We are now using `QStringLiteral()` to wrap string literals (e.g. QStringLiteral("subject_rich")) in new code.

Reply:
Thanks, changed the strings in this function to QStringLiterals